databus/src/main/java/com/bazaarvoice/emodb/databus/core/Canary.java

```diff
@@ -78,7 +78,7 @@ private void createCanarySubscription() {
         // the subscription at startup.  The subscription should last basically forever.
         // Note: make sure that we don't ignore any events
         _databus.subscribe(_subscriptionName, _subscriptionCondition,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "subscribe"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)